src/main/java/com/example/weather/client/controller/WeatherDataController.java

```diff
@@ -0,0 +1,26 @@
+package com.example.weather.client.controller;
+
+import com.example.weather.client.models.dto.WeatherDataDto;
+import com.example.weather.client.services.WeatherDataService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/weather/data")
+public class WeatherDataController {
+
+    private final WeatherDataService weatherDataService;
+
+    @GetMapping
+    public Page<WeatherDataDto> getWeatherDataByCity(
+            @RequestParam(defaultValue = "Warsaw") String city,
```

Comment:
```suggestion
            @RequestParam String city,
```
default value here doesn't really make sense, I was talking just about the other two